code/lib/blocks/src/components/Story.tsx

```diff
@@ -113,8 +115,23 @@ const IFrameStory: FunctionComponent<IFrameStoryProps> = ({ story, height = '500
  * A story element, either rendered inline or in an iframe,
  * with configurable height.
  */
+
+const ErrorMessage = styled.strong(({ theme }) => ({
+  color: theme.color.orange,
+}));
+
 const Story: FunctionComponent<StoryProps> = (props) => {
-  const { inline } = props;
+  const { inline, story } = props;
+
+  if (inline && !props.autoplay && story.usesMount) {
+    return (
+      <ErrorMessage>
+        This story mounts inside of play. Set{' '}
+        <a href="https://storybook.js.org/docs/api/doc-blocks/doc-block-story#autoplay">autoplay</a>{' '}
```

Comment:
Make sure to track the docs update as a followup issue

Reply:
https://github.com/storybookjs/storybook/issues/28442

Reply:
As an alternative, we might just *not* filter these stores in `Stories` and instead use this message? Would that solve the discoverability issue?

The user could just set `tags: ['-autoplay']` on the story to avoid the error, no?

Reply:
I could live with that, although the error is quite a bit ugly atm,  cc @shilman